features/step_definitions/direct_award_steps.rb

```diff
@@ -1,9 +1,14 @@
 When(/^I have created and saved a search$/) do
   steps %Q{
-    Given I am on the /g-cloud/search page
+    Given I am on the /g-cloud/search?q=test+123 page
```

Comment:
"test 123" is a search with zero results (at least with data as it currently stands) - feels like there's a danger here that we never test downloading an ODS file with any rows in it - might be better to pick base search terms that will return some results.

There is then the problem of ensuring that the final "download results" isn't too big... `/g-cloud/search?q=email+analysis+provider` has 19 results on prod at the moment, could work?

Also, do we want an extra test for where a user can't download too many results (e.g. search for `software` with no filters and check the expected error message appears)?

Reply:
Thanks @TheDoubleK - Will update the query.

Regarding the "too many results" flow. We have tested this in the unit tests. So I think that will be fine.